components/molecule/modal/src/index.scss

```diff
@@ -0,0 +1,109 @@
+@import '~@schibstedspain/sui-theme/lib/settings-compat-v7/index';
+@import '~@schibstedspain/sui-theme/lib/index';
+
+body.is-modal-open {
+  overflow-y: hidden;
+}
+
+.sui-MoleculeModal  {
+  background-color: $bg-modal-basic;
+  bottom: 0;
+  display: none;
+  left: 0;
+  position: fixed;
+  right: 0;
+  text-align: center;
+  top: 0;
+  white-space: nowrap;
+  z-index: $z-modal-basic;
+
+  &.is-open {
+    display: flex;
+    justify-content: center;
+    overflow-x: hidden;
+    overflow-y: auto;
+  }
+
+  &--verticallyCentered {
+    align-items: center;
+  }
+
+  &-dialog {
+    background-color: $bg-modal-basic-dialog;
+    border: $bd-modal-basic;
+    box-shadow: $bxsh-modal-basic;
+    display: flex;
+    flex-direction: column;
+    margin: $m-modal-basic;
+    max-height: 100%;
+    text-align: left;
+    vertical-align: middle;
+    white-space: normal;
+
+    @include media-breakpoint-down(s) {
```

Comment:
we need to remove this media query, more info https://github.schibsted.io/scmspain/frontend-all-wiki/wiki/Breakpoints